index.js

```diff
@@ -1,3 +1,4 @@
+require('babel-plugin-check-es2015-constants');
```

Comment:
@jakepusateri, `babel-plugin-check-es2015-constants` will check all consts, to ensure them readOnly.
example:
```
const count = 3
count = 4 // Throw an error in chrome 48: Assignment to constant variable
```

If `babel-plugin-check-es2015-constants` was not require(), this example would pass in babel, and `const` may be transformed to `var`.

Add this line, babel will throw an error:
```
SyntaxError: test.js: "a" is read-only
  1 | const a = 3
> 2 | a = 4
    | ^
  3 | 
```